nifi-extension-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/AbstractJMSProcessor.java

```diff
@@ -128,6 +128,14 @@ public abstract class AbstractJMSProcessor<T extends JMSWorker> extends Abstract
             .identifiesControllerService(JMSConnectionFactoryProviderDefinition.class)
             .build();
 
+    static final PropertyDescriptor MAX_BATCH_SIZE = new PropertyDescriptor.Builder()
+            .name("Maximum Batch Size")
+            .description("The number of messages to publish or consume in a single iteration of the processor.")
```

Comment:
Thanks for making batch size consistent in these processors.
NiFi does not have for loops in the ui so I suggest using "invocation" instead of "iteration" if you don't mind.

Reply:
I fixed the wording, and also stealthily made the same change to the GetFile Batch Size description.